test/index.js

```diff
@@ -6,44 +6,37 @@ try {
   process.exit(1)
 }
 
+var Buffer = require('safe-buffer').Buffer
 var OpenSSL_EVP_BytesToKey = require('bindings')('OpenSSL_EVP_BytesToKey')
 var crypto = require('crypto')
 var test = require('tape')
 var EVP_BytesToKey = require('../')
 
-function createHashMD5 () { return crypto.createHash('md5') }
 var keyLen = 32
 var ivLen = 16
 
-var counts = [1, 2, 100]
-for (var i = 0; i < counts.length; ++i) {
-  var count = counts[i]
+for (var i = 0; i < 1000; ++i) {
+  var password = crypto.randomBytes(10 + Math.round(Math.random() * 100))
+  var salt = crypto.randomBytes(8)
```

Comment:
the 'fuzzing' style tests can be pretty helpful to find weird edge cases, that being said I'm not against also doing some deterministic tests as well